mp_locales/en-US.json

```
@@ -93,6 +93,9 @@
     "ui.multiplayer.official": "Official Servers",
     "ui.multiplayer.servers": "Public Servers",
     "ui.multiplayer.favorites": "Favorites",
+    "ui.multiplayer.favorite": "Save as Favorite",
+    "ui.multiplayer.favorited_toast_title": "Favorited server",
+    "ui.multiplayer.favorited_toast_description": "This server has been favorited.",
```

```suggestion
    "ui.multiplayer.favoritedToast.title": "Favorited server",
    "ui.multiplayer.favoritedToast.description": "This server has been favorited.",
```
Suggestion to keep the casing consistent